ui/package.json

```diff
@@ -96,7 +96,8 @@
     "shebang-loader": "^0.0.1",
     "snake-case": "^3.0.3",
     "subscriptions-transport-ws": "^0.11.0",
-    "taskcluster-client-web": "^60.4.2",
+    "taskcluster-client": "68.0.4",
+    "taskcluster-client-web": "68.0.4",
```

Comment:
any idea why this is needed (why it doesn't just come from source directory), and why it was on such an old version?

Reply:
oh, thanks for spotting! was trying to upgrade client and added the wrong one. Removing.

The other one is not linked directly most likely because we deploy docs to netlify, maybe that complicated linking? I'm not sure to be honest